GcmEndpoints/README.md

```diff
@@ -7,7 +7,9 @@ This backend template employs [Google Cloud Endpoints](https://developers.google
 
 # 1. Adding a backend in Android Studio
 
-To add the backend to your existing Android app from this backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
+Start with an existing Android app. You can use the [GCM sample Quickstart app](https://github.com/googlesamples/google-services/tree/master/android/gcm) as a starting point.
+
```

Comment:
Yeah, I think it's best to delegate to the GCM maintainers when talking about the client side GCM setup. I think our tutorial should limit itself, as far as possible, to the appengine related setup.

I'm not really sure if we need to ask them to jump at any point though, given that we've removed all the android gcm client setup.